src/main/java/org/jenkinsci/plugins/gitclient/verifier/KnownHostsFileVerifier.java

```diff
@@ -66,8 +66,8 @@ public boolean verifyServerHostKey(String hostname, int port, String serverHostK
 
     private void logHint(TaskListener listener) {
         listener.getLogger().println(HyperlinkNote.encodeTo("https://plugins.jenkins.io/git-client/#plugin-content-ssh-host-key-verification/","You're using 'Known host file' strategy to verify ssh host keys," +
-                " but your known_hosts file not exists, please go to " +
-                "‘Manage Jenkins’ -> ‘Configure Global Security’ -> ‘Git Host Key Verification Configuration' " +
+                " but your known_hosts file does not exist, please go to " +
+                "‘Manage Jenkins’ -> ‘Configure Global Security’ -> ‘Git Host Key Verification Configuration‘ " +
```

Comment:
Should there be non-ASCII "funny quotes" emitted here? I think standard single-quotes can suffice to keep things simple?

Reply:
let's use simple quotes to be safe